docs/reference/api.md

```diff
@@ -1332,6 +1332,7 @@ A utility builder to be used with `fcl.args[...]` to create FCL supported argume
 
 ```javascript
 import * as fcl from "@onflow/fcl";
+import * as types from "@onflow/types";
```

Comment:
Types are exposed in FCL now, so should be able to remove this line and use `fcl.t` for types.

Reply:
Ah, even better! I updated the examples accordingly.